Mathlib/Tactic/NormNum/Ineq.lean

```diff
@@ -216,7 +242,7 @@ where
   | .isRat _ .., _ | _, .isRat _ .. => ratArm
   | .isNegNat _ .., _ | _, .isNegNat _ .. => intArm
   | .isNat ra na pa, .isNat rb nb pb =>
-    let _i ← inferOrderedSemiring α
+    let ⟨_, _, _i⟩ ← inferOrderedSemiring α
```

Comment:
It's not a job for this PR but one wonders whether this stuff can be tidied up now, maybe we're inferring more stuff than we need and then throwing it away here. Maybe we want `inferIsOrderedSemiring` instead or something? Maybe add a TODO near the top of the file of the form "consider tidying up now `OrderedSemiring` has been replaced by `IsOrderedSemiring`" or something? Or maybe I'm fussing about nothing.